2021-02-24 15:02:22|DEBUG|succeed to upload file to UCloud_Engine_Local|https://www.qingwazb.com/upload///thumb/20210224/06674941429817714.png
2021-02-24 16:22:37|ERROR|Login.AnchorLogin|PDOException: Unknown column 'user_money' in 'field list' in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(811): NotORM_Result->query('SELECT issuper,...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(887): NotORM_Result->execute()
#2 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(875): NotORM_Result->fetchRow('')
#3 /www/wwwroot/yunbao.com/PhalApi/Appapi/Model/Login.php(17): NotORM_Result->fetchOne()
#4 /www/wwwroot/yunbao.com/PhalApi/Appapi/Domain/Login.php(9): Model_Login->anchorLogin('16666666666', '###c90b6949ca29...')
#5 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/Login.php(89): Domain_Login->anchorLogin('16666666666', 'w123456')
#6 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_Login->anchorLogin()
#7 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#8 {main}
2021-02-24 20:16:38|ERROR|User.SetCash|PDOException: Unknown column 'NAN' in 'field list' in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(410): NotORM_Result->query('UPDATE cmf_user...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/Appapi/Model/User.php(293): NotORM_Result->update(Array)
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Domain/User.php(72): Model_User->setCash(Array)
#3 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/User.php(1956): Domain_User->setCash(Array)
#4 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_User->setCash()
#5 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#6 {main}
2021-02-24 20:19:55|ERROR|User.SetCash|PDOException: Unknown column 'NAN' in 'field list' in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(410): NotORM_Result->query('UPDATE cmf_user...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/Appapi/Model/User.php(293): NotORM_Result->update(Array)
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Domain/User.php(72): Model_User->setCash(Array)
#3 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/User.php(1956): Domain_User->setCash(Array)
#4 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_User->setCash()
#5 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#6 {main}
2021-02-24 22:24:36|ERROR|Family.FamilyStatistics|PDOException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''Array' and addtime >= 1614096000 and addtime <= 1614182400)' at line 1 in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(748): NotORM_Result->query('SELECT SUM(tota...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(790): NotORM_Result->aggregation('SUM(totalcoin)')
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/Family.php(65): NotORM_Result->sum('totalcoin')
#3 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_Family->familyStatistics()
#4 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#5 {main}
2021-02-24 22:24:43|ERROR|Family.FamilyStatistics|PDOException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''Array' and addtime >= 1614096000 and addtime <= 1614182400)' at line 1 in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(748): NotORM_Result->query('SELECT SUM(tota...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(790): NotORM_Result->aggregation('SUM(totalcoin)')
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/Family.php(65): NotORM_Result->sum('totalcoin')
#3 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_Family->familyStatistics()
#4 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#5 {main}
2021-02-24 22:33:42|ERROR|Family.FamilyStatistics|PDOException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''(42885,109328)' and addtime >= 1614096000 and addtime <= 1614182400)' at line 1 in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(748): NotORM_Result->query('SELECT SUM(tota...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(790): NotORM_Result->aggregation('SUM(totalcoin)')
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/Family.php(68): NotORM_Result->sum('totalcoin')
#3 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_Family->familyStatistics()
#4 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#5 {main}
2021-02-24 22:34:43|ERROR|Family.FamilyStatistics|PDOException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''\"(42885,109328)\"' and addtime >= 1614096000 and addtime <= 1614182400)' at line 1 in /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php:258
Stack trace:
#0 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(748): NotORM_Result->query('SELECT SUM(tota...', Array)
#1 /www/wwwroot/yunbao.com/PhalApi/PhalApi/NotORM/NotORM/Result.php(790): NotORM_Result->aggregation('SUM(totalcoin)')
#2 /www/wwwroot/yunbao.com/PhalApi/Appapi/Api/Family.php(68): NotORM_Result->sum('totalcoin')
#3 /www/wwwroot/yunbao.com/PhalApi/PhalApi/PhalApi.php(57): Api_Family->familyStatistics()
#4 /www/wwwroot/yunbao.com/public/appapi/index.php(15): PhalApi->response()
#5 {main}
